### PR TITLE
Add FolderBrowserDialog tests

### PR DESCRIPTION
**Pull request by hughbe**

### CODE REVIEW

File: src/System.Windows.Forms/src/System/Windows/Forms/FolderBrowserDialog.cs [hunk 1]
```diff
@@ -2,188 +2,121 @@
...
+    [DefaultEvent(nameof(HelpRequest))]
+    [DefaultProperty(nameof(SelectedPath))]
+    [Designer("System.Windows.Forms.Design.FolderBrowserDialogDesigner, " + AssemblyRef.SystemDesign),]
+    [SRDescription(nameof(SR.DescriptionFolderBrowserDialog))]
     public sealed class FolderBrowserDialog : CommonDialog
     {
         // Root node of the tree view.
-        private Environment.SpecialFolder rootFolder;
-    
+        private Environment.SpecialFolder _rootFolder;
```
Comment (zsd4yr) [inline src/System.Windows.Forms/src/System/Windows/Forms/FolderBrowserDialog.cs]: not cleanup, change of name. Can you separate out this PR a bit more?

Comment (hughbe) [reply src/System.Windows.Forms/src/System/Windows/Forms/FolderBrowserDialog.cs]: Sure - fixed

**Comment by danmosemsft** [comment]
@zsd4yr this seems like it can be merged now.

**Comment by danmosemsft** [comment]
@zsd4yr for changes like this organized into nice commits - what is your repo preference - to rebase, merge, or squash? I think where there are big cleanups, it is probably better to rebase or merge to keep them separate - and otherwise squash. LMK.

**Comment by RussKie** [comment]
Please don't squash. Rebase, if necessary, and merge as is

**Pull request merged**